packages/core/src/components/Tooltip/Tooltip.tsx

```diff
@@ -100,7 +103,7 @@ interface TooltipBaseProps extends VibeComponentProps {
    */
   showOnDialogEnter?: boolean;
   /**
-   * A Classname to be added to <spam> element which wraps the children
+   * A Classname to be added to <span> element which wraps the children
```

Comment:
didn't you fixed it already on a different PR? 🤔

Reply:
I also remember such thing haha